### PR TITLE
fix style syntax for mir.ndslice

### PR DESCRIPTION
**Pull request by wilzbach**
Pending changes for ndslice from #88

### CODE REVIEW

File: source/mir/ndslice/iteration.d [hunk 1]
```diff
@@ -127,12 +127,10 @@ template swapped(size_t dimensionA, size_t dimensionB)
     auto swapped(size_t N, Range)(Slice!(N, Range) slice)
     {
         {
-            enum i = 0;
             alias dimension = dimensionA;
             mixin DimensionCTError;
         }
         {
-            enum i = 1;
```
Comment (9il) [inline source/mir/ndslice/iteration.d:135]: BTW, have you looked at `DimensionCTError`?

Comment (wilzbach) [inline source/mir/ndslice/iteration.d:135]: damn - i just looked at the first one. My mistake :/

```
    static assert(dimension < N,
        "dimension = " ~ dimension.stringof ~ " at position "
        ~ i.stringof ~ " should be less than N = " ~ N.stringof
        ~ tailErrorMessage!());
```

**Comment by 9il** [comment]
i can not merge this until phobos PR, but i can merge fixes for sum and combinatorics

**Comment by wilzbach** [comment]
I understood, #88 only contains those. Dscanner will only print a couple of warnings which we can resolve as soon as ndslice is open for changes.

**Comment by wilzbach** [comment]
This is exactly similar to #154 - closing in favor of the newer PR.

**Pull request closed without merging**